eng/workloads/OptionalWorkloads.vsmanproj

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="VSSetup.targets" />
+
+  <PropertyGroup>
+    <DebugSymbols>false</DebugSymbols>
+    <IsShippingAssembly>false</IsShippingAssembly>
+    <PublishWindowsPdb>false</PublishWindowsPdb>
+    <TargetType>build-manifest</TargetType>
+    <FinalizeManifest>true</FinalizeManifest>
+    <FinalizeSkipLayout>false</FinalizeSkipLayout>
+    <ProductName>DotNetOptionalWorkloads</ProductName>
+    <ProductFamily>vs</ProductFamily>
+    <ProductFamilyVersion Condition="$(ProductFamilyVersion) == ''">42.42.42</ProductFamilyVersion>
```

Comment:
I assume we need to pass in the real ProductFamilyVersion somewhere, I don't see it in this PR.

Reply:
Correct, this may need to run during staging, since the actual build needs to extract data from the signed MSIs when creating merge manifests